server.py

```diff
@@ -4,10 +4,9 @@
 path = os.path.abspath(__file__)
 dir_path = os.path.dirname(path)
 
-
-
 app = Bottle()
 
+expected = open('./files/expected_outputs/expected_output.txt').read()
```

Comment:
```suggestion
with open('./files/expected_outputs/expected_output.txt', 'r') as fl:
    expected = fl.read()
```

The other thing leaves the file open which is fine in this case but will lead to a bad habit later on.